config/settings.yml

```diff
@@ -1299,6 +1301,7 @@
       :session_timeout_interval: 30.seconds
       :storage_file_collection_interval: 1.days
       :storage_file_collection_time_utc: 21600
+      :task_timeout_check_frequency: 1.hour
```

Comment:
`1.hour` here seems reasonable. We don't want to be checking too often but, we also don't want to have tasks sitting in an active state for an extended period of time when they are already dead.